metadata.yaml

```diff
@@ -40,11 +46,16 @@ requires:
   certificates:
     interface: tls-certificates
     limit: 1
+    optional: true
   s3-parameters:
     interface: s3
     limit: 1
+    optional: true
 
 storage:
   pgdata:
     type: filesystem
-    location: /var/lib/postgresql/data
```

Comment:
Looks like this is a mountpoint for the charm sidecar container. We should either be `container.push`ing resouces (`patroni.yaml`) or restore this.